### DISCUSSION TITLE
Records support

### DISCUSSION THREAD

**Discussion by listepo** [medz/dart-orm#305] [Q&A] [answered]
will it be possible to generate only records instead of classes?

**Comment by medz** [comment, marked as answer]
@listepo Good idea, that's what I meant. Yes, support is planned via `prisma.user.findMany().unserialized();`! I'm opening this as an issue as a feature request in 4.x.

**Reply by medz** [reply]
The unserialized method is supported in the latest 4.0.0-alpha.1 version, use it as in the answer above!
<img width="738" alt="image" src="https://github.com/medz/prisma-dart/assets/5564821/9909920f-1cb2-4ac2-9b4e-fb419e7e4a70">
